build-lib/util/licenser.js

```diff
@@ -25,7 +25,7 @@ Licenser.prototype.constructor = Licenser;
 
 Licenser.prototype.build = function () {
 
-  const fileDescriptions = [].concat(...this.inputPaths.map(dirname => {
```

Comment:
bumped to support at least 4.4.7, spread operator became available natively after that.